src/components/header.scss

```diff
@@ -0,0 +1,21 @@
+nav {
+  border-bottom: 3px solid #000;
+  ul{
```

Comment:
Use whitespaces and extra lines for readability please.
You can try to add a linter for css.